crates/subspace-farmer/src/bin/subspace-farmer/commands/farm.rs

```diff
@@ -82,19 +62,39 @@ pub(crate) async fn farm(
     })
     .await??;
 
-    let subspace_codec = SubspaceCodec::new(identity.public_key());
+    // TODO: Piece count should account for database overhead of various additional databases
+    // For now assume 80% will go for plot itself
+    let plot_size = plot_size * 4 / 5 / PIECE_SIZE as u64;
```

Comment:
Another TODO here is to remember previously specified plot size and delete old plots if upon restart smaller size was specified.